py/selenium/webdriver/remote/remote_connection.py

```diff
@@ -368,6 +368,8 @@ def __init__(self, remote_server_addr, keep_alive=False, resolve_ip=True):
                 ('POST', '/session/$sessionId/window/rect'),
             Command.GET_WINDOW_RECT:
                 ('GET', '/session/$sessionId/window/rect'),
+            Command.W3C_MINIMIZE_WINDOW:
```

Comment:
I closed the #4272 because  the AppVeyor build failed. So I thought maybe the commit should be more simply. So I removed the MINIMIZE_WINDOW part. However it still failed.
So I want to know the #4272 commit is OK or not. If OK, I will push it again and add a test for it.
THX!